targets/docker/README.md

```diff
@@ -0,0 +1,37 @@
+# `Dockerfile`
+
+# run build
+
+Run from the top-level directory:
+
+```sh
+jerryscript $ ls -ah
+.               .gitignore      CONTRIBUTING.md LICENSE         README.md       docs            jerry-libm      tests
+..              .travis.yml     DCO.md          LOGO.png        cmake           jerry-core      jerry-main      third-party
+.git            CMakeLists.txt  Doxyfile        LOGO.svg        docker          jerry-libc      targets         tools
+jerryscript $ ./targets/docker/build.sh
+```
+
+## output
+
+```sh
+=== Summary ===
+ - Ran 11552 tests
+ - Passed 11546 tests (99.9%)
+ - Failed 6 tests (0.1%)
+
+Failed tests
```

Comment:
Are those tests still failing when running the JerryScript test suite with with Docker?